src/CommunityToolkit.Maui.Core/Views/Popup/MauiPopup.tizen.cs

```diff
@@ -0,0 +1,145 @@
+using NHorizontalAlignment = Tizen.NUI.HorizontalAlignment;
+using NVerticalAlignment = Tizen.NUI.VerticalAlignment;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The native implementation of Popup control.
+/// </summary>
+public class MauiPopup : Popup
+{
+	readonly IMauiContext mauiContext;
+
+	/// <summary>
+	/// Constructor of <see cref="MauiPopup"/>.
+	/// </summary>
+	/// <param name="mauiContext">An instance of <see cref="IMauiContext"/>.</param>
+	/// <exception cref="ArgumentNullException">If <paramref name="mauiContext"/> is null an exception will be thrown. </exception>
+	public MauiPopup(IMauiContext mauiContext)
+	{
+		this.mauiContext = mauiContext ?? throw new ArgumentNullException(nameof(mauiContext));
+		OutsideClicked += OnOutsideClicked;
+	}
+
+	/// <inheritdoc/>
+	protected override void Dispose(bool isDisposing)
+	{
+		if (isDisposing)
+		{
+			OutsideClicked -= OnOutsideClicked;
+		}
+
+		base.Dispose(isDisposing);
+	}
+
+	void OnOutsideClicked(object? sender, EventArgs e)
+	{
+		if (VirtualView is null || VirtualView.Handler is null)
+		{
+			return;
+		}
+
+		if (VirtualView.CanBeDismissedByTappingOutsideOfPopup)
+		{
+			Close();
+			VirtualView.Handler.Invoke(nameof(IPopup.OnDismissedByTappingOutsideOfPopup));
+		}
+	}
+
+	/// <summary>
+	/// An instance of the <see cref="IPopup"/>.
+	/// </summary>
+	public IPopup? VirtualView { get; private set; }
+
+	/// <summary>
+	/// Method to initialize the native implementation.
+	/// </summary>
+	/// <param name="element">An instance of <see cref="IPopup"/>.</param>
+	public void SetElement(IPopup? element)
+	{
+		VirtualView = element;
+	}
+
+
+	/// <summary>
+	/// Method to show the Popup
+	/// </summary>
+	public void ShowPopup()
+	{
+		_ = VirtualView ?? throw new InvalidOperationException($"{nameof(VirtualView)} cannot be null");
+		Content = VirtualView.Content?.ToPlatform(mauiContext) ?? throw new InvalidOperationException($"{nameof(VirtualView.Content)} cannot be null");
+
+		if (Content is null)
+		{
+			return;
+		}
+
+		BackgroundColor = new Tizen.NUI.Color(0.1f, 0.1f, 0.1f, 0.5f);
+		Content.BackgroundColor = (VirtualView.Color ?? Colors.Transparent).ToNUIColor();
+
+		if (VirtualView.Anchor is not null)
+		{
+			var anchorView = VirtualView.Anchor.ToPlatform();
+			var anchorPosition = anchorView.ScreenPosition;
+			Content.ScreenToLocal(out float x, out float y, anchorPosition.X, anchorPosition.Y);
+			Layout = new AbsoluteLayout();
+			Content.UpdatePosition(new Tizen.UIExtensions.Common.Point(x, y));
+		}
+		else
+		{
+			Layout = new LinearLayout
+			{
+				LinearOrientation = LinearLayout.Orientation.Vertical,
+				VerticalAlignment = ToVerticalAlignment(VirtualView.VerticalOptions),
+				HorizontalAlignment = ToHorizontalAlignment(VirtualView.HorizontalOptions),
+			};
+			Content.UpdatePosition(new Tizen.UIExtensions.Common.Point(0, 0));
+		}
+
+		UpdateContentSize();
+
+		Open();
+		VirtualView?.OnOpened();
```

Comment:
Virtual can't be null here, there's a check at line 73

```suggestion
		VirtualView.OnOpened();
```

Reply:
Done